tests/containers/rootless_podman.pm

```diff
@@ -164,19 +164,26 @@ sub verify_userid_on_container {
 
 }
 
-sub check_bsc1200623() {
-    my ($cid) = shift;
+sub check_bsc1200623 {
     # When this bug appears, the output (cid) is composed of 2 lines, e.g.
     # cid = "
     # 2022-07-05T08:48:56.151176-04:00 susetest systemd[3438]: Failed to start podman-6734.scope.
     # 5b08b0dc136dd32bb30e69e4deb5df511dea0602d6b0c8d3623120370184506a"
-    # So, we need to remove the first one.
-    if ($cid =~ /Failed to start podman/) {
-        record_soft_failure('bsc#1200623 - systemd[3557]: Failed to start podman-3627.scope') unless ($bsc1200623);
-        $bsc1200623 = 1;    # to prevent printing the soft-failure more than once
-        ($cid) =~ s/.*\n//;
+    # So, we need to remove everything except the container ID.
+    my $input = shift;
+
+    if ($input =~ /([a-f0-9]{64})/) {
+        my $cid = $1;
```

Comment:
Sorry, in your solution to me is not clear why the 2 separated inputs
my $input = shift;
then
my $cid = $1;

Neither clear what output those should contain.

In the header's failing test, `script_output('podman run...` seems mixed stdout + stderr. So first I'd separate the 2 output on 2 files, then check those.

The use cases should be 3 at least:
1) all ok = no out printed,
2) generic failure => some generic check and reaction done,
3) bsc occurred => specific bsc check and reaction done.

Reply:
There's no stderr.  It's console output from the s390x backend mixed with stdout.

The simple solution is to sanitize `$input` and compare against the expected hexadecimal container ID.

Reply:
what should $1 in cid contian, then?

Reply:
...when the bsc occur?

Reply:
> what should $1 in cid contain, then?

The container ID: 64 hexadecimal characters.

Reply:
Ok, I mean, could `($cid ne $input)` be possible to happen for non-$bsc1200623 errors also ?